clusterloader2/pkg/measurement/common/probes/probes.go

```diff
@@ -37,8 +37,8 @@ const (
 
 	manifestsPathPrefix = "$GOPATH/src/k8s.io/perf-tests/clusterloader2/pkg/measurement/common/probes/manifests/"
 
-	checkProbesReadyInterval = 15 * time.Second
-	checkProbesReadyTimeout  = 5 * time.Minute
+	defaultCheckProbesReadyInterval = 15 * time.Second
```

Comment:
Let's not expose the interval as a var - timeout seems to be enough for now.

Reply:
Make sense. We don't need it and we can easily expose it if needed.